src/gui/search/searchsortmodel.cpp

```diff
@@ -47,8 +119,47 @@ bool SearchSortModel::lessThan(const QModelIndex &left, const QModelIndex &right
         if (Utils::String::naturalSort(vL.toString(), vR.toString(), res))
             return res;
 
-        return QSortFilterProxyModel::lessThan(left, right);
+        return base::lessThan(left, right);
+    }
+
+    return base::lessThan(left, right);
+}
+
+bool SearchSortModel::filterAcceptsRow(int source_row, const QModelIndex &source_parent) const
```

Comment:
I meant that the function signature and parameter names are part of Qt code, not qBittorrent. The overrides will become hardly untraceable if everyone change parameters names. Or I can add `override` keyword. Then the fact of overriding will be explicit.